src/main/java/net/earthcomputer/clientcommands/UnsafeUtils.java

```diff
@@ -0,0 +1,34 @@
+package net.earthcomputer.clientcommands;
+
+import sun.misc.Unsafe;
+
+import java.lang.invoke.MethodHandles;
+import java.lang.reflect.Field;
+
+/**
+ * @author Gaming32
+ */
+public class UnsafeUtils {
+    private static final Unsafe UNSAFE;
+    private static final MethodHandles.Lookup IMPL_LOOKUP;
+
+    static {
+        try {
+            final Field unsafeField = Unsafe.class.getDeclaredField("theUnsafe");
+            unsafeField.setAccessible(true);
+            UNSAFE = (Unsafe) unsafeField.get(null);
+            final Field implLookupField = MethodHandles.Lookup.class.getDeclaredField("IMPL_LOOKUP");
+            IMPL_LOOKUP = (MethodHandles.Lookup) UNSAFE.getObject(UNSAFE.staticFieldBase(implLookupField), UNSAFE.staticFieldOffset(implLookupField));
+        } catch (Exception e) {
+            throw new RuntimeException(e);
```

Comment:
This should probably more gracefully fail, and disable the command. It could possibly fail if the user is using a different JVM such as OpenJ9, or in future Java versions.

I'm not super keen on having this hack in here at all, but for now I've decided it's fine.